### ISSUE TITLE
Leaving the project

### ISSUE THREAD

**Issue by wimpunk** [ddclient/ddclient#380]
Hi All,

I'm not sure who's actually still reading the issues but at least I don't have the time anymore to work on this project.  Each year again since the last 5 years I've planned to work on it again but it didn't worked out.  I became dad twice and my children and wife need the time I used to spend on ddclient. 
So if there is anyone else who wants to take over the project, please let me know.  Maybe @SuperSandro2000 is still active, but according to the last commit, even that was a while ago.

It's been fun but now I really have to leave this project.

**Comment by wimpunk** [comment]
Maybe I should also mention @rhansen and @DaveSophoServices who have been active before.  Maybe they want to take full control on this project.

**Comment by bitshiftnetau** [comment]
> Hi All,
> 
> I'm not sure who's actually still reading the issues but at least I don't have the time anymore to work on this project. Each year again since the last 5 years I've planned to work on it again but it didn't worked out. I became dad twice and my children and wife need the time I used to spend on ddclient. So if there is anyone else who wants to take over the project, please let me know. Maybe @SuperSandro2000 is still active, but according to the last commit, even that was a while ago.
> 
> It's been fun but now I really have to leave this project.

Congratulations dude! I'm sure someone will take over and do a great job.

**Comment by SuperSandro2000** [comment]
I am still active on GitHub with just shy over 19000 contributions in 2021 but I don't really have the time right now to really code anything for ddclient. The software is working for me right now and I am not motivated to add any new obscure and sometimes questionable services. Also some of the ones included right now I would rather remove to reduce the complexity of the program.

If you want you can give me owner rights in the organization and I can try to manage new contributors and try to keep it alive but I don't think I will code much.

**Comment by DaveSophoServices** [comment]
@supersandro2000

There are a few bug fixes that need to be merged in. I might be able to handle those, or anyone could propose a pull  request to master. 

@wimpunk, kids are a lot of work and it’s great that you’re supporting you wife in this, and not just letting her take care of the kids alone.

**Comment by wimpunk** [comment]
@SuperSandro2000 You're promoted to owner. Let me know if you're still missing something.

**Comment by reetp** [comment]
Hi,

@wimpunk well done for all your work.

Open source is a challenge at the best of times.

But family first. Enjoy your new family. It all goes by far to fast!!

@SuperSandro2000 

I do a bit of work at contribs.org with Koozali SME server and we use ddclient in a contrib.

I've just been doing some work on it myself and was debating whether we should fork & maintain our own version of ddclient.

I'm no great coder but am happy to put my hand up to try & assist with maintenance here.

A few in our team may assist aa well.

Let me know.

**Comment by TerryFage** [comment]
Like reetp do a little testing etc on Koozali SME, happy to also throw the hat in to help maintain ddclient..

**Comment by SuperSandro2000** [comment]
> I could give you access to sf.net if you give me your login name.

https://sourceforge.net/u/supersandro2000/profile

> About the domain: feel free to take it over. It's paid for a while (not sure until when) but afterward it can be taken over.

I think it would be better if we do it in a coordinated way and not let anyone else squat it.

**Comment by DarthGandalf** [comment]
Yeah. Depending on the registrar it should be easy enough to transfer the domain to Sandro.

**Comment by dkerr64** [comment]
Thank you @wimpunk for all your contributions and being honest about not having the time to commit to the project.  Just starting this topic allows us all to think about how to get ddclient back to a well maintained project.

**Comment by wimpunk** [comment]
> > I could give you access to sf.net if you give me your login name.
> 
> https://sourceforge.net/u/supersandro2000/profile

Fixed, added you as admin.

**Comment by rrthomas** [comment]
@wimpunk, is there anything left to resolve with this issue?

**Comment by LenardHess** [comment]
We're still in the process of handing over the domain.
(semi-related: @rrthomas can you unarchive the docs repo?)

Looking through this, the sourceforge page and the mailing list are a thing I didn't consider so far.

**Comment by rrthomas** [comment]
> (semi-related: @rrthomas can you unarchive the docs repo?)

Done.

**Comment by rrthomas** [comment]
I'd be very happy to archive (or whatever is possible) the SF page and end the mailing list, in each case with a suitable notice redirecting to GitHub.
